File: ejercicio_036/ejercicio_36.py
n = int(input("Ingresa el tamaño de la lista (impar): "))
numeros = []
print("Ingresa los números: ")
for i in range(n):
    numero = int(input())
    numeros.append(numero)
unico = 0 
for i in range(n):
    count = 0 
    for j in range(n):
        if numeros[i] == numeros[j]:
            count += 1
    if count == 1:
        unico = numeros[i]
        break 
print("El número único es: ", unico)

File: ejercicio_036/ejercicio_36.java
package programacion_practica.ejercicio_36;
import java.util.Scanner;
public class ejercicio_36 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingresa el tamaño de la matriz (impar): ");
        int n = scanner.nextInt();
        int[] numeros = new int[n];
        System.out.println("Ingresa los números:");
        for (int i = 0; i < n; i++) {
            numeros[i] = scanner.nextInt();
        }
        int unico = 0;
        for (int i = 0; i < n; i++) {
            int count = 0;
            for (int j = 0; j < n; j++) {
                if (numeros[i] == numeros[j]) {
                    count++;
                }
            }
            if (count == 1) {
                unico = numeros[i];
                break;
            }
        }
        System.out.println("El número único es: " + unico);
        scanner.close();
    }
}
